#,Feature,TestCase,Result,ResultType,ExecutedOn,Description,Function,Module,Devices,KnownIssue
1,Regression,FtOpSoSwLagFn042,Pass,Mapped,2024-01-06 03:33:06,Test case passed,test_ft_portchannel_with_vlan_variations,switching/test_portchannel.py,"V3, V4",
2,Regression,FtOpSoSwLagFn041,Pass,Mapped,2024-01-06 03:33:06,Test case passed,test_ft_portchannel_with_vlan_variations,switching/test_portchannel.py,"V3, V4",
3,PortChannel Config Enhancements,FtOpSoSwPominFn007,Pass,Mapped,2024-01-06 03:33:27,Test case passed,test_ft_verify_min_links_functionality_by_flapping_all_member_ports_007,switching/test_portchannel.py,"V3, V4",
4,Regression,ft_lldp_LocManAddrLen,Fail,Mapped,2024-01-06 03:37:08,module configuration failed. LLDP neighbors information is not found after polling @5445,test_ft_lldp_LocManAddrLen,system/test_lldp.py,"V3, V4",
5,Regression,ft_lldp_LocManAddrlfld,Fail,Mapped,2024-01-06 03:37:09,module configuration failed. LLDP neighbors information is not found after polling @5445,test_ft_lldp_LocManAddrlfld,system/test_lldp.py,"V3, V4",
6,Regression,ft_lldp_LocManAddrEntry,Fail,Mapped,2024-01-06 03:37:09,module configuration failed. LLDP neighbors information is not found after polling @5445,test_ft_lldp_LocManAddrEntry,system/test_lldp.py,"V3, V4",
7,Regression,ft_lldp_ConfigManAddrEntry,Fail,Mapped,2024-01-06 03:37:09,module configuration failed. LLDP neighbors information is not found after polling @5445,test_ft_lldp_ConfigManAddrEntry,system/test_lldp.py,"V3, V4",
8,Regression,ft_lldp_lldplocportid,Fail,Mapped,2024-01-06 03:37:10,module configuration failed. LLDP neighbors information is not found after polling @5445,test_ft_lldp_lldplocportid,system/test_lldp.py,"V3, V4",
9,Regression,ft_lldp_lldplocsysname,Fail,Mapped,2024-01-06 03:37:10,module configuration failed. LLDP neighbors information is not found after polling @5445,test_ft_lldp_lldplocsysname,system/test_lldp.py,"V3, V4",
10,Regression,ft_lldp_lldplocsysdesc,Fail,Mapped,2024-01-06 03:37:10,module configuration failed. LLDP neighbors information is not found after polling @5445,test_ft_lldp_lldplocsysdesc,system/test_lldp.py,"V3, V4",
11,SNMP Agent Optimization,ft_snmp_lldp_prem_manaddr_table,Fail,Mapped,2024-01-06 03:37:11,module configuration failed. LLDP neighbors information is not found after polling @5445,test_ft_lldp_rem_man_addr_table,system/test_lldp.py,"V3, V4",
12,Regression,FtOpSoSwlldpFn011,Fail,Mapped,2024-01-06 03:37:11,module configuration failed. LLDP neighbors information is not found after polling @5445,test_ft_lldp_non_default_config,system/test_lldp.py,"V3, V4",
13,Regression,FtOpSoSwlldpFn010,Fail,Mapped,2024-01-06 03:37:11,module configuration failed. LLDP neighbors information is not found after polling @5445,test_ft_lldp_non_default_config,system/test_lldp.py,"V3, V4",
14,Regression,FtOpSoSwlldpFn009,Fail,Mapped,2024-01-06 03:37:11,module configuration failed. LLDP neighbors information is not found after polling @5445,test_ft_lldp_non_default_config,system/test_lldp.py,"V3, V4",
15,Regression,FtOpSoSwlldpFn008,Fail,Mapped,2024-01-06 03:37:11,module configuration failed. LLDP neighbors information is not found after polling @5445,test_ft_lldp_non_default_config,system/test_lldp.py,"V3, V4",
16,Regression,FtOpSoSwlldpFn004,Fail,Mapped,2024-01-06 03:37:11,module configuration failed. LLDP neighbors information is not found after polling @5445,test_ft_lldp_non_default_config,system/test_lldp.py,"V3, V4",
17,Regression,FtOpSoSwlldpFn013,Fail,Mapped,2024-01-06 03:37:12,module configuration failed. LLDP neighbors information is not found after polling @5445,test_ft_lldp_docker_restart,system/test_lldp.py,"V3, V4",
18,Regression,lldp_verify_counters,Fail,Mapped,2024-01-06 03:37:12,module configuration failed. LLDP neighbors information is not found after polling @5445,test_lldp_verify_counters,system/test_lldp.py,"V3, V4",
19,Regression,ft_lldp_with_remote_server,Fail,Mapped,2024-01-06 03:37:12,module configuration failed. LLDP neighbors information is not found after polling @5445,test_ft_lldp_with_remote_server,system/test_lldp.py,"V3, V4",
20,Regression,ft_ssh_service_disable,Pass,Mapped,2024-01-06 03:39:45,Test case passed,test_ft_ssh_service_disable,system/test_ssh.py,"V3, V4",
21,Control Plane ACL,test_ft_controlplane_acl_service_sshv6,EnvFail,Mapped,2024-01-06 03:52:51,Console hang observed. Exiting.,test_ft_ssh_add_user_verify,system/test_ssh.py,"V3, V4",
22,Control Plane ACL,test_ft_controlplane_acl_service_sshv4,EnvFail,Mapped,2024-01-06 03:52:51,Console hang observed. Exiting.,test_ft_ssh_add_user_verify,system/test_ssh.py,"V3, V4",
23,Control Plane ACL,test_ft_controlplane_acl_service_snmp,EnvFail,Mapped,2024-01-06 03:52:51,Console hang observed. Exiting.,test_ft_ssh_add_user_verify,system/test_ssh.py,"V3, V4",
24,Regression,ft_ssh_add_user_verify,EnvFail,Mapped,2024-01-06 03:52:51,Console hang observed. Exiting.,test_ft_ssh_add_user_verify,system/test_ssh.py,"V3, V4",
25,Regression,CETA_SONIC_67575,ConfigFail,Mapped,2024-01-06 05:08:32,module configuration failed.  @5445,test_ft_verify_logs_at_the_time_of_ssh_login_success,system/test_ssh.py,"V3, V4",
